computer/src/rpc.rs

```diff
@@ -0,0 +1,96 @@
+use std::fmt;
+use std::sync::{Arc, Mutex};
```

Comment:
I think we should use tokio Mutex https://doc.servo.org/tokio/sync/struct.Mutex.html since it does not throw error when thread is locked

Reply:
Done https://github.com/openrankprotocol/openrank/pull/220/commits/824b348a1ca90778bdd81e75bdcbda1144e11b44